packages/neuron-wallet/src/database/chain/ormconfig.ts

```diff
@@ -133,28 +144,54 @@ const connectOptions = async (genesisBlockHash: string): Promise<SqliteConnectio
       IndexerTxHashCacheRemoveField1701234043431,
       CreateCellLocalInfo1701234043432,
     ],
+    subscribers: [
+      AddressSubscribe,
+      AssetAccountSubscribe,
+      MultisigConfigSubscribe,
+      SudtTokenInfoSubscribe,
+      TxDescriptionSubscribe,
+    ],
     logger: 'simple-console',
     logging,
     maxQueryExecutionTime: 30,
   }
 }
 
-export const initConnection = async (genesisBlockHash: string) => {
+const initConnectionWithType = async (genesisBlockHash: string, connectionName: ConnectionName) => {
   // try to close connection, if not exist, will throw ConnectionNotFoundError when call getConnection()
   try {
-    await getConnection().close()
+    await getConnection(connectionName).close()
   } catch (err) {
     // do nothing
   }
-  const connectionOptions = await connectOptions(genesisBlockHash)
+  const connectionOptions = await connectOptions(genesisBlockHash, connectionName)
 
   try {
     await createConnection(connectionOptions)
-    await getConnection().manager.query(`PRAGMA busy_timeout = 3000;`)
-    await getConnection().manager.query(`PRAGMA temp_store = MEMORY;`)
+    await getConnection(connectionName).manager.query(`PRAGMA busy_timeout = 3000;`)
+    await getConnection(connectionName).manager.query(`PRAGMA temp_store = MEMORY;`)
   } catch (err) {
     logger.error(err.message)
   }
 }
 
+export async function initConnection(genesisBlockHash: string) {
+  await initConnectionWithType(genesisBlockHash, 'full')
+  await initConnectionWithType(genesisBlockHash, 'light')
+}
+
+export function migrateDBFile(genesisBlockHash: string) {
```

Comment:
Recommend renaming this method to a more certain name to ensure that it will work if a later version also needs to be migrated, such as

```ts
declare function migrate_0_112(): Promise<void>;

function migrate() {
  migrate_0_112()
}
```

And in the future version, we can upgrade it by

```ts
function migrate() {
  if (lastVersion <= v0_112) await migrate_0_112()
  if (lastVersion <= v0_11x) await migrate_0_11x()
}
```

Reply:
Currently, we do not save the last version. If we need to use the last version, we'd better save the last version in the next version. And then in that next version we can get the last version

Reply:
Does the comment above help @homura

Reply:
Repost